backend/src/models/room.ts

```diff
@@ -0,0 +1,7 @@
+import Video from "./video";
+
+export default interface Room {
+  name: string;
+  url: string;
```

Comment:
let's change the url field to be currVideo or currVideoId instead so it better reflects what it is